packages/terra-form-textarea/docs/MigrationGuide.md

```diff
@@ -0,0 +1,19 @@
+# Migrating from terra-form's textarea Component to terra-form-textarea
```

Comment:
Add:
* examples of the old usage vs new usage.
* Add old & new prop tables.

Reply:
Removed migration guide. https://github.com/cerner/terra-core/pull/982/commits/802777623b56e1099bdf699936a4de4e2853d6ff